<?xml version="1.0"?>
<launch>
    <!-- This is a atlasmv launch files for starting a planar_pc_generator from a laser scan-->
    <group ns="/pcp/fus/planar_pc">

        <remap from="/laserscan0" to="/scan"/>
        <remap from="/tracking_frame" to="/map"/>
        <remap from="/pc_out" to="/pointcloud"/>

        <node name="planar_pc" pkg="mtt" type="simple_planar_pc_generator_atlasmv" output="screen">
            <param name="output_frequency" value="100.0"/>
            <param name="perpendicular_threshold" value="0.15"/>
        </node>

    </group>
</launch>